misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -3687,12 +3697,51 @@ public OrderKeyNode transform(OrderKeyNode orderKeyNode) {
                 .apply();
     }
 
+    @Override
+    public ClassDefinitionNode transform(ClassDefinitionNode classDefinitionNode) {
+        if (!isInLineRange(classDefinitionNode, lineRange)) {
+            return classDefinitionNode;
+        }
+
+        Token visibilityQualifier = getToken(classDefinitionNode.visibilityQualifier().orElse(null));
+        MetadataNode metadata = this.modifyNode(classDefinitionNode.metadata().orElse(null));
+        NodeList<Token> qualifierList = this.modifyNodeList(classDefinitionNode.classTypeQualifiers());
+        Token classKeyword = getToken(classDefinitionNode.classKeyword());
+        Token className = getToken(classDefinitionNode.className());
+        NodeList<Node> members = this.modifyNodeList(classDefinitionNode.members());
+        Token openBrace = getToken(classDefinitionNode.openBrace());
+        Token closeBrace = getToken(classDefinitionNode.closeBrace());
+        int startColumn = getStartColumn(classDefinitionNode, true);
+
+        if (metadata != null) {
+            classDefinitionNode = classDefinitionNode.modify()
+                    .withMetadata(metadata).apply();
+        }
+
+        if (visibilityQualifier != null && !visibilityQualifier.isMissing()) {
+            classDefinitionNode = classDefinitionNode.modify()
+                    .withVisibilityQualifier(formatToken(visibilityQualifier, startColumn, 1, 0, 0))
+                    .apply();
+            startColumn = 0;
+        }
+
+        return classDefinitionNode.modify()
+                .withClassKeyword(formatToken(classKeyword, startColumn, 0, 0, 0))
+                .withClassName(formatToken(className, 1, 0, 0, 0))
+                .withOpenBrace(formatToken(openBrace, 1, 0, 0, 1))
+                .withCloseBrace(formatToken(closeBrace, 0, 0, 0, 1))
+                .withClassTypeQualifiers(qualifierList)
+                .withMembers(members)
+                .apply();
+    }
+
     @Override
     public Token transform(Token token) {
         int startColumn = getStartColumn(token, true);
         if (token.kind() == (SyntaxKind.COMMA_TOKEN)) {
             return formatToken(token, 0, 1, 0, 0);
-        } else if (token.kind() == (SyntaxKind.PUBLIC_KEYWORD) || token.kind() == (SyntaxKind.ABSTRACT_KEYWORD)) {
+        } else if (token.kind() == (SyntaxKind.PUBLIC_KEYWORD) || token.kind() == (SyntaxKind.ABSTRACT_KEYWORD) ||
```

Comment:
since we return within each of the clauses, shall we use if clauses

Reply:
Updated.